dev/format_python_code.sh

```diff
@@ -37,6 +37,7 @@ for paths in \
     "cvat/apps/events/signals.py" \
     "cvat/apps/engine/management/commands/syncperiodicjobs.py" \
     "cvat/apps/dataset_manager/management/commands/cleanuplegacyexportcache.py" \
+    "cvat/apps/redis_handler/**/*.py" \
```

Comment:
```suggestion
    "cvat/apps/redis_handler/" \
```

bash doesn't know about the `**` syntax.